docs/gas-schedule-proposal.md

```diff
@@ -0,0 +1,223 @@
+| A1 | LOG1 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| A2 | LOG2 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| A3 | LOG3 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| A4 | LOG4 | static_gas = 375 <br/> dynamic_gas = 375 * topic_count + 8 * size + memory_expansion_cost | static_gas = 35 <br/> dynamic_gas = 35 * topic_count + 8 * size + memory_expansion_cost |
+| F1 | CALL | address_access_cost = 100 (warm address) | address_access_cost = 150 (warm address) |
+| F4 | DELEGATECALL | address_access_cost = 100 (warm address) | address_access_cost = 150 (warm address) |
+| FA | STATICCALL | address_access_cost = 100 (warm address) | address_access_cost = 150 (warm address) |
+| F3 | RETURN | static_gas = 0 <br/> dynamic_gas = memory_expansion_cost | static_gas = 5 <br/>dynamic_gas = memory_expansion_cost |
+| FD | REVERT | static_gas = 0 <br/> dynamic_gas = memory_expansion_cost | static_gas = 8 <br/>dynamic_gas = memory_expansion_cost |
+
+
+| Precompile | Name | Current Gas | Proposed Gas |
+| ------------- | ------------- | ------------- | ------------- |
+| 01 | ECRECOVER | 3000 | 17000 |
+| 06 | ECADD | 150 | 3500 |
+| 07 | ECMUL | 6000 | 10000 |
+| 0A | POINTEVAL | 50000 | 400000 |
+
+
+## Radical Gas Schedule Proposal
+The idea behind the radical gas schedule proposal is a complete overhaul of the current gas schedule. Rather then just changing the most mispriced opcodes, we propose to change all opcodes to better reflect the computational cost of the operations.
+
+Let's run through the consequences of the radical gas schedule proposal. The cheapest operations are priced at 1 gas and we gradually increase the gas cost for more complex operations. As a result most arithmetic and basic opcodes will be much cheaper, i.e. valued at 1 rather than 3 or 5. Then all other operations will be adjusted accordingly, usually by lowering the gas cost. 
+
+In this scenario the storage cost remains at the same level as this refects the network cost of storing data. Thus the radical gas schedule proposal expands the gap between the cost of storage and computation. This is a good thing as it makes it more expensive to store data than to compute it. Also the memory expansion cost remains the same to increase the security of the network.
+
+Pros:
+- The gas cost reflects the true computational cost of the operations
+- The larger gap between the cost of storage and computation promotes more efficient use of the network
+- Configurable gas schedules are easier update in the future
```

Comment:
Configurable gas schedules? What do you mean? This therm was not used yet. Some clarification would be suitable here.